# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/NeckoChannelParams.ipdlh
# Commit: effee4ab270c
# Full Hash: effee4ab270cba85ad53b7be844212bc458ec1a6
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2019-10-03 04:38:57
# Regressor Bug: 1584207
# File Overlap Count: 1
# Description:
#   Bug 1584207 - Forward nsILoadInfo.requestBlockingReason to the parent process for both old and new redirect channels, r=dragana,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D47264
# ==============================================================================

diff -r a49132352f5b -r effee4ab270c netwerk/ipc/NeckoChannelParams.ipdlh
--- a/netwerk/ipc/NeckoChannelParams.ipdlh	Wed Oct 02 20:53:50 2019 +0200
+++ b/netwerk/ipc/NeckoChannelParams.ipdlh	Tue Oct 01 16:50:23 2019 +0000
@@ -213,6 +213,8 @@
   // The ServiceWorker controller may be cleared in the child during
   // a redirect.
   IPCServiceWorkerDescriptor? controller;
+
+  uint32_t requestBlockingReason;
 };
 
 //-----------------------------------------------------------------------------